assets/namespace.d.ts

```diff
@@ -1,6 +1,3 @@
-// This file was overwritten by prisma-json-types-generator
-// Report any issues to https://github.com/arthurfiorette/prisma-json-types-generator
```

Comment:
Why remove this warning? I like having it so people remember to open issues here and not on prisma.

Reply:
The warning is still present in the modified files: https://github.com/arthurfiorette/prisma-json-types-generator/pull/477/commits/ca4ae2ad1b345b3da57054c050c04e7633c86670#diff-1216bea603a5c17ffdb36f4cf391e21d29cd3ef4940ce453fac91121158fc707
It was removed from that file because modified model files no longer included the `namespace.d.ts` file verbatim (they instead import it). Should I also add a comment to the generated `pjtg.ts` file clarifying it was produced by `prisma-json-types-generator`?